hooks/tk-multi-publish2/ingest/ingest_files.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import os
+import copy
+import glob
+import pprint
+import traceback
+
+import sgtk
+from sgtk import TankError
+
+from tank.util.shotgun import get_sg_connection
+
+HookBaseClass = sgtk.get_hook_baseclass()
+
+
+class IngestFilesPlugin(HookBaseClass):
+    """
+    Inherits from PublishFilesPlugin
+    """
+
+    @property
+    def description(self):
+        """
+        Verbose, multi-line description of what the plugin does. This can
+        contain simple html for formatting.
+        """
+
+        desc = super(IngestFilesPlugin, self).description
```

Comment:
The parent description is describing a publish process, which (even though this class inherits from) is not an accurate description for what is happening here. Since this is user-facing, please make sure you fully describe (so that an artist or producer can understand) exactly what this process is doing.

Reply:
Updated this.